### PR TITLE
Avatar group component

### LINKED ISSUE #1778: Create an "Avatar group" component
### Description

The goal of this task is to create an Avatar group component, stacking multiple avatars together, with a predefined spacing between single avatars.

### Figma link

_No response_

### How urgent this is?

It's blocking me right now

### PR DESCRIPTION
**Pull request by drillprop**
Fix #1778

### CODE REVIEW

**Review by rafalpawlow — CHANGES_REQUESTED** [review]
I think they have an incorrect offset e.g In figma `40px` variant has `8px` offset but you added only `4px`. Or I don't understand figma measurements correctly
Zrzut ekranu 2021-12-28 o 20 41 38

**Comment by rafalpawlow** [comment]
Also labels are not centered correctly
Zrzut ekranu 2021-12-28 o 20 44 54

File: src/components/Avatar/AvatarGroup.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  component: AvatarGroup,
+  argTypes: {
+    size: {
+      control: {
+        type: 'select',
+        options: ['default', 'bid', 'small'],
+      },
+      defaultValue: 'default',
+    },
+    clickable: {
```
Comment (rafalpawlow) [inline src/components/Avatar/AvatarGroup.stories.tsx]:
```suggestion
    clickable: { type: 'boolean', defaultValue: true }
```

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

**Review by rafalpawlow — APPROVED** [review]

File: src/components/Avatar/AvatarGroup.styles.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+`
+
+export const AvatarOverlay = styled.div<{ dimmed?: boolean }>`
+  pointer-events: none;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  position: absolute;
+  top: 0;
+  background-color: black;
```
Comment (rafalpawlow) [inline src/components/Avatar/AvatarGroup.styles.ts:47]: I think this black should come from variable

**Comment by toiletgranny** [comment]
Everything seems to be fine, hurray! 🎉 Just two quick questions/notes:
1. Is my understanding correct that the color of the outline of each avatar is meant to be a variable that can be easily overridden?
2. Is it just me, or the size labels on Storybook are broken? 😅 `default`, `bid`, `small`?

**Review by toiletgranny — APPROVED** [review]

**Pull request merged**